market/solutions/onboarding/README.md

```diff
@@ -0,0 +1,26 @@
+Axon Ivy’s Employee Onboarding solution gives HR the best practice guidance to coordinate responsibilities to ensure a smooth provision of work material and other arrangements. Moreover, HR manager and the superior can capture all relevant information which is necessary to automatically create the employee in the HR system. This solution:
+
+- ensures a warm welcome to new employees from day one (and even the days before)
+- guarantees no tasks left undone
+- promises new hires become productive much faster
+- makes sure that needed materials can be purchased in advance
+- increases retention rates and finally
+- contributes to a good reputation for the company’s image
+
+## At a Glance
```

Comment:
@andreasbalsiger  ???  a product with neither an installer nor an information where to get it?